docs/docs/feature-library/queues-view-data-tiles.md

```diff
@@ -0,0 +1,124 @@
+
+### Queues View Columns
+This feature also includes the option to show two additional columns in the Queues Stats View: Assigned Tasks and Wrapping Tasks. Again, the Active task count column is the sum of Assigned Tasks and Wrapping Tasks. Displaying these metrics separately may give Supervisors additional insight into how their agents are performing.
+
+# Setup
+
+This feature can be enabled via the `flex-config` attributes. Just set the `queues_view_data_tiles` `enabled` flag to `true` and set up the desired configuration.
+
+* To enable specific data tiles set `_DataTile = true`
+* You can change the Channel colors are needed. 
+* The Enhanced Agent Activity tile replaces the native Bar Chart so if you enable it you can disable the Bar Chart.
+* Configure activities to match the names of the Activities as defined in TaskRouter. The Flex UI includes a [set of icons](https://www.twilio.com/docs/flex/developer/ui/v1/icons#default-icons)
+ that are used to enhance the display of the individual activities.
+* In the `queuesStatsColumns`, enable the additional queue metrics (assigned & wrapping tasks) to display.
+
+```json
+   "queues_view_data_tiles": {
+        "enabled": true,
+        "activeTasksDataTile": false,
+        "waitingTasksDataTile": false,
+        "longestWaitTimeDataTile": false,
+        "agentsByActivityBarChart": false,
+        "channels": {
+          "voice": {
+            "color": "#ADD8E6",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          },
+          "chat": {
+            "color": "#87CEFA",
+            "SLADataTile": false,
+            "taskCountsDataTile": false
+          },
+          "sms": {
+            "color": "#4682B4",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          }
+        },
+        "allChannelsDataTile": true,
+        "enhancedAgentByActivityPieChart": true,
+        "agentActivityConfiguration": {
+          "Available": {
+            "color": "green",
+            "icon": "Accept"
+          },
+          "Outbound": {
+            "color": "greenyellow",
+            "icon": "Call"
+          },
+          "Break": {
+            "color": "goldenrod",
+            "icon": "Hold"
+          },
+          "Lunch": {
+            "color": "darkorange",
+            "icon": "Hamburger"
+          },
+          "Training": {
+            "color": "red",
+            "icon": "Bulb"
+          },
+          "OTHER": {
+            "color": "darkred",
+            "icon": "More"
+          },
```

Comment:
Can you update this with the new schema?

Reply:
Done